cmake_modules/AllPythonBindings.cmake

```diff
@@ -136,9 +135,9 @@ if(tests)
 endif(tests)
 
 
-
+# Follow conda convention for Python installation in Windows 
 if(WIN32)
-  set(PYINSTALL_DIRECTORY dcprogs/python-pkg)
+  set(PYINSTALL_DIRECTORY ${PYTHON_INTERP_PREFIX}/lib/site-packages)
 else()
```

Comment:
I don't think this is the right way. PYINSTALL_DIRECTORY should be a relative directory and then CMAKE_INSTALL_PREFIX should be set to ${PYTHON_INTERP_PREFIX} if not explicitly set to something else